frameworks/D/hunt/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Clean any files from last run
```

Comment:
@viile the `travis.yml` file you need to add to is in the project root, not the framework root. This is what runs all the tests. Also, don't add any languages to that `travis.yml` file. You will need to setup your dependencies as @tchaloupka suggested. Travis is meant to verify the framework works in our environments.